crates/node/src/cli/sync.rs

```diff
@@ -48,6 +48,10 @@ pub struct SyncParams {
     /// Periodically create a backup, for debugging purposes. Use it with `--backup-dir <PATH>`.
     #[clap(long, value_name = "NUMBER OF BLOCKS")]
     pub backup_every_n_blocks: Option<u64>,
+
+    /// Ignore the order of the blocks to allow starting at some height.
+    #[clap(long)]
+    pub ignore_block_order: bool,
```

Comment:
i'd prefer if this was implied when using --starting-block, because i can't see any other case where you would want --starting-block without --ignore-block-order

Reply:
also, i think we should put a huge warning with starting-block saying that it'll definitely break your db. Maybe rename --starting-block to --unsafe-starting-block, because that may be a bit surprising if the user provides --starting-block expecting some kind of fork-testing behavior and getting a totally broken db
and maybe even hide the option in clap hidden because I don't want users to actually use it lol